### PR TITLE
HPND: add markup

### LINKED ISSUE #1725: Add XML markup: HPND (for pngcheck) [SPDX-Online-Tools]
**1.** License Name: pngcheck
**2.** Short identifier: pngcheck
**3.** License Author or steward: Unknown
**4.** Comments: This is the license of the pngcheck utility. The license appears to date to the 1990's, and has been stable for many years. The pngcheck utility is packaged in a number of Linux distributions (https://repology.org/project/pngcheck/packages), which suggests it should be considered a significant project. It is an extremely permissive license similar to NTP-0, but with different phrasing. Upstream describes it as an “MIT/X11-style license,” which seems accurate. Fedora Linux has found it to be an acceptable open source license in https://gitlab.com/fedora/legal/fedora-license-data/-/issues/85.
**5.** License Request Url: http://tools.spdx.org/app/license_requests/187
**6.** URL(s): http://www.libpng.org/pub/png/src/pngcheck-3.0.3.LICENSE
**7.** OSI Status: Not Submitted
**8.** Example Projects: http://www.libpng.org/pub/png/apps/pngcheck.html

### PR DESCRIPTION
**Pull request by jlovejoy**
Fixes #1725 

to be able to use "It... " or "This software..." 

Signed-off-by: Jilayne Lovejoy

### CODE REVIEW

**Comment by jlovejoy** [comment]
ugh! why is this failing?

**Comment by goneall** [comment]
> ugh! why is this failing?

@jlovejoy - here's the error message:

`File ./src/HPND.xml: Error: Opening and ending tag mismatch: alt line 39 and a`

File: src/HPND.xml [hunk 1]
```diff
@@ -33,9 +33,11 @@
             <alt match=".*" name="copyrightHolder0">&lt;copyright holder&gt;</alt>
             <alt match=".*" name="orRelated">&lt;or related entities&gt;</alt>
          not be used in advertising or publicity pertaining to distribution of the software without specific, written
-         prior permission</optional>. <optional>
-            <alt match=".*" name="copyrightHolder1">&lt;copyright holder&gt;</alt> makes no representations
-         about the suitability of this software for any purpose. It is provided "as is" without express or implied warranty.</optional>
+         prior permission</optional>. 
+         <optional><alt match=".*" name="copyrightHolder1">&lt;copyright holder&gt;</alt> 
+         makes no representations about the suitability of this software for any purpose. 
+         <alt match="It|This software" name="subject">It</a>
```
Comment (goneall) [inline src/HPND.xml:39]: @jlovejoy You're missing a `/` after `"subject"`.  `<alt match="It|This software" name="subject">` -> `<alt match="It|This software" name="subject"/>`

Comment (jlovejoy) [reply src/HPND.xml:39]: huh, but the other alt tags don't have that?

Comment (swinslow) [reply src/HPND.xml:39]: I think the problem is that the opening tag is `<alt>` but the closing tag is `</a>`, the latter probably needs to be `</alt>`

**Pull request merged**